engine/src/flutter/lib/web_ui/test/engine/semantics/semantics_helper_test.dart

```diff
@@ -64,6 +64,24 @@ void testMain() {
       }
     });
 
+    test('Tab key should not enable semantics', () async {
+      final testSemanticsEnabler = FakeSemanticsEnabler();
+
+      // Tab should not enable semantics
+      {
+        final event = createDomKeyboardEvent('keydown', <String, Object>{'key': 'Tab'});
+        expect(testSemanticsEnabler.shouldEnableSemantics(event), isTrue);
```

Comment:
"Tab should not enable semantics" -> expect(shouldEnableSemantics, isTrue)

Massively confused by this API :P

Reply:
I know! Me too.